tests/ga/test_extension.py

```diff
@@ -682,120 +673,6 @@ def test_it_should_process_valid_extensions_if_present(self, mock_get, mock_cryp
             expected_handlers.remove(handler.name)
         self.assertEqual(0, len(expected_handlers), "All handlers not reported status")
 
-    def test_ext_zip_file_packages_removed_in_update_case(self, *args):
```

Comment:
The ZIP package is now deleted after expansion so all these tests are not needed

Reply:
yup, i should have added that test.

added now: TestWireClient.test_download_zip_package_should_expand_and_delete_the_package